jui-core/index.d.ts

```diff
@@ -7,6 +7,10 @@
 
 declare var jui: JuiStatic;
 
+declare type ClassFunction = (() => any);
```

Comment:
I removed ClassFunction, VoidFunction and AnyFunction. 

but i redefined correctly  parameters and  return types.